### ISSUE TITLE
r-catboost conda-forge needs latest version

### ISSUE THREAD

**Issue by ashleyabraham** [catboost/catboost#1254]
Problem: r-catboost version is older in  conda-forge 
catboost version: 0.20.2
Operating System: Ubuntu 16.04.6 LTS
CPU: Intel(R) Xeon(R) CPU E5-2699 v4 @ 2.20GHz
GPU: GPU 0: Tesla P100-PCIE-16GB

I am using catboost in R, installed using conda  r-catboost 0.20.2 from conda-forge and the GPU is supported, I was able to run using the GPU, at least up until I get the error that was fixed in [Issue 1072](https://github.com/catboost/catboost/issues/1072). So to get the latest, I installed r-catboost 0.22 directly from source according to instructions  
`devtools::install_url('https://github.com/catboost/catboost/releases/download/v0.22/catboost-R-Linux-0.22.tgz', INSTALL_opts = c("--no-multiarch"))`
or `devtools::install_github('catboost/catboost', subdir = 'catboost/R-package')` but the GPU/CUDA error,  I get the following error in 0.22 and not in 0.20.2
```
Error in catboost.train(pool, test.pool, params = fit_params) :
  catboost/cuda/cuda_lib/cuda_base.h:281: CUDA error 35: CUDA driver version is insufficient for CUDA runtime version
Execution halted
```
Can you fix the r-catboost  0.22 package with GPU support and also update conda-forge

**Comment by kizill** [comment]
Hello! We've switched cuda toolkit to 10.1, so now you need nvidia driver version >= 418.39. Could you update driver?

**Comment by andrey-khropov** [comment]
The issue with version updates has been fixed and the newest release 0.23 is now available in conda but you'll still need to update your GPU driver in order to use newer versions of CatBoost.
